src/components/DocContainer.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import styled from 'styled-components/native';
+import DocHolder, { Missing, Submitted } from 'components/DocHolder';
+import { TextBold } from 'assets/fonts/Fonts';
+import { Colors } from 'assets/Colors';
+
+
+interface Props { 
+    caseType: string;
+    uploadStatus: boolean; //based on all submissions
+}
+
+
+const Container = styled.View`
```

Comment:
Move all styles into a seperate file and import these styled components! See `screens/Welcome`for example :)